code/game/objects/items/robot/robot_items/robot_inflatable_dispenser.dm

```diff
@@ -10,36 +11,54 @@
 
 	var/list/stored_walls = list()
 	var/list/stored_doors = list()
+	var/list/stored_shelters = list()
 	var/max_walls = 4
 	var/max_doors = 3
-	var/list/allowed_types = list(/obj/item/inflatable/wall, /obj/item/inflatable/door)
+	var/max_shelters = 0
```

Comment:
>CE isn't allowed to deploy shelters but borgs are

Reply:
the request was for borgs :man_shrugging: